core/exception/signm_spec.rb

```diff
@@ -1,5 +1,10 @@
 require_relative '../../spec_helper'
 
 describe "SignalException#signm" do
-  it "needs to be reviewed for spec completeness"
+  it "returns the signal name" do
+    name = Signal.list.keys.last
+    -> { Process.kill(name, Process.pid) }.should raise_error(SignalException) { |e|
```

Comment:
I'm not sure this works. In any case it seems best to use a well-known signal like SIGINT:
```
$ ruby -e 'begin; Process.kill(:INT, Process.pid); rescue SignalException => e; p e.class.ancestors; end'
[Interrupt, SignalException, Exception, Object, Kernel, BasicObject]
```
Same below.

Reply:
I could not make this work with INT, so used TERM.

Reply:
Thanks!